### ISSUE TITLE
Game crash before it starting

### ISSUE THREAD

**Issue by xpxpert** [yifanlu/PSXperia#28]
Hello, How you doin :)

I have problem that when i open the game it says updating and that's it, it doesn't work at all
Xperia Play - Android 2.3.3

this is the steps

Last login: Fri Sep 16 00:21:57 on ttys000
/Users/Isa/Downloads/psxperia-beta-2/psxperiatool-gui ; exit;
Isa-Alhammadis-iMac:~ Isa$ /Users/Isa/Downloads/psxperia-beta-2/psxperiatool-gui ; exit;
[I] Starting extraction with PSXPeria Extractor version 0.3 Beta 3
[I] Verifying files
[I] Decoding values
Sep 16, 2011 12:24:35 AM brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loading resource table...
Sep 16, 2011 12:24:35 AM brut.androlib.res.AndrolibResources loadMainPkg
INFO: Loaded.
Sep 16, 2011 12:24:35 AM brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loading resource table from file: /Users/Isa/apktool/framework/1.apk
Sep 16, 2011 12:24:37 AM brut.androlib.res.AndrolibResources loadFrameworkPkg
INFO: Loaded.
Sep 16, 2011 12:24:37 AM brut.androlib.res.AndrolibResources decode
INFO: Decoding file-resources...
Sep 16, 2011 12:24:37 AM brut.androlib.res.AndrolibResources decode
INFO: Decoding values_/_ XMLs...
Sep 16, 2011 12:24:37 AM brut.androlib.res.AndrolibResources decode
INFO: Done.
[I] Extracting APK
[I] Extracting ZIP file: /Users/Isa/Desktop/untitled folder 13/com.sony.playstation.ncea00344_1.apk to: /Users/Isa/Desktop/P1
[D] Done extracting ZIP.
[I] Extracting ZPAKS
[I] Extracting ZIP file: /Users/Isa/Desktop/P1/assets/NCEA00344_1_1.zpak to: /Users/Isa/Desktop/P1/assets/ZPAK
[D] Done extracting ZIP.
[I] Extracting ZIP file: /Users/Isa/Desktop/untitled folder 13/NCEA00344_1_1.zpak to: /Users/Isa/Desktop/P1/ZPAK
[D] Done extracting ZIP.
[I] Removing unneeded files.
[I] Adding new files.
[I] Patching XML strings with tags.
[D] Replacing strings in XML data.
[D] Writing to temporary file: /Users/Isa/Desktop/P1/assets/AndroidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: /Users/Isa/Desktop/P1/assets/ZPAK/metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: /Users/Isa/Desktop/P1/res/values/strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: /Users/Isa/Desktop/P1/ZPAK/metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[I] Done.
[I] PSXperiaTool initialized, outputting to: /Users/Isa/Desktop/Crash 3
[I] Starting build with PSXPeria Converter version 0.3 Beta 3
[I] Checking to make sure all files are there.
[D] Done checking data.
[I] Creating temporary directory.
[D] Created temporary directory at, ./.psxperia.530
[I] Copying icon if needed.
[D] Done copying icon from /Users/Isa/Desktop/untitled folder 13/images.png
[I] Replacing strings.
[D] Replacing strings in XML data.
[D] Writing to temporary file: ./.psxperia.530/assets/AndroidManifest.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ./.psxperia.530/assets/ZPAK/metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ./.psxperia.530/res/values/strings.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Writing to temporary file: ./.psxperia.530/ZPAK/metadata.xml.tmp
[D] Successfully cleaned up and done with string replacement.
[D] Done with string replacement.
[D] Done replacing strings.
[I] Generating PSImage.
[I] Beginning PSImage compression.
[I] Generating image TOC.
[D] Done generating PSImage
[I] Generating ZPAK.
[I] Generating zpak file from directory ./.psxperia.530/ZPAK with compression = false
[D] Done with ZPAK creation.
[D] Done generating ZPAK at ./.psxperia.530/NCXA12345_1.zpak
[I] Generating default ZPAK.
[I] Generating zpak file from directory ./.psxperia.530/assets/ZPAK with compression = true
[D] Done with ZPAK creation.
[D] Done generating default ZPAK at ./.psxperia.530/assets/NCXA12345_1.zpak
[I] Done processing, generating output.
[D] Moving files around.
[D] Running command: [./aapt-osx package -f -F /Users/Isa/Desktop/Crash 3/com.sony.playstation.NCXA12345_1.apk.unsigned -S ./.psxperia.530/res -M ./.psxperia.530/assets/AndroidManifest.xml -I ./android-framework.jar ./.psxperia.530]
[I] Signing apk /Users/Isa/Desktop/Crash 3/com.sony.playstation.NCXA12345_1.apk.unsigned to /Users/Isa/Desktop/Crash 3/com.sony.playstation.NCXA12345_1.apk
[I] Done.
[I] APK file: /Users/Isa/Desktop/Crash 3/com.sony.playstation.NCXA12345_1.apk
[I] Data dir: /Users/Isa/Desktop/Crash 3/data/com.sony.playstation.NCXA12345_1/files/content
[I] Deleting temporary directory
[I] Done.

hope i can find a solution

**Comment by yifanlu** [comment]
Try the troubleshooting tips on the wiki.

**Comment by yifanlu** [comment]
From the lack of response, I assume the issue is fixed.

**Comment by guywithcomp** [comment]
Im sorry, I am having the same issue as xpxpert. I use the Xperia Play - Android 2.3.3 . I ripped my Resident Evil 3 to .iso and tested it on the psx emulator which works fine. I use PSXperia version .1 because the Beta 2 gives me a 404 error, but the apk is created with no issues. I put the apk on the phone and the data files in Android/data, install it and the game shows up but when I run the apk, it tries to update and brings me to the home screen. I was going to post my own topic, but this is the same issue I assume. Any guidance would be appreciated. Also I tried the troubleshooting tips on the wiki, but no use. Thank you so much.
